src/ios/IRoot.m

```diff
@@ -136,28 +136,29 @@ - (bool) jailbroken {
 
         (f = fopen("/bin/bash", "r")) ||
         (f = fopen("/bin/sh", "r")) ||
-        (f = fopen("/usr/sbin/sshd", "r")) ||
```

Comment:
@Zach-Attach Why we should remove this check?

Reply:
@WuglyakBolgoink Sorry, that wasn't intentional. I saw "/usr/bin/sshd" and thought it was a duplicate.

Reply:
@WuglyakBolgoink It is a duplicate. See line 116

Reply:
ah. ok.